webapp/channels/src/components/new_search/search_hint.tsx

```diff
@@ -98,10 +104,12 @@ const SearchHints = ({onSelectFilter, searchType, searchTerms, searchTeam, hasSe
     return (
         <SearchHintsContainer id='searchHints'>
             <i className='icon icon-lightbulb-outline'/>
-            <FormattedMessage
-                id='search_hint.filter'
-                defaultMessage='Filter your search with:'
-            />
+            <h2>
```

Comment:
I know that we were asked to add a heading here, but that feels wrong since these options correspond to the search box above this. 0/5 on if we want to revert this change and argue about that or leave it as-is though

Reply:
I meant that the new heading moves the buttons into their own section of the page, separate from the search box which those buttons control, and that's what feels wrong to me

Reply:
@hmhealey if not a heading, what element do think makes more sense then?

Reply:
I'm having a hard time finding anything that matches the visible UX here, and I can't think of any examples of other apps that use a similar pattern at the moment (Slack and Discord both offer those flags as autocomplete results). Looking at just HTML elements, the closest thing I can find is a [`fieldset`](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/fieldset) wrapping the "Filter your search with" text and buttons with the text in a `legend` element, but that doesn't group them with the search box which also seems incorrect.

As it stands, I think it's best to go with Level Access's suggestion for now since I can't find a better way of representing this which matches our visible UX